packages/angular_devkit/build_angular/src/tools/webpack/configs/styles.ts

```diff
@@ -96,6 +96,23 @@ export async function getStylesConfig(wco: WebpackConfigOptions): Promise<Config
       extraPostcssPlugins.push(require(tailwindPackagePath)({ config: tailwindConfigPath }));
     }
   }
+  // Check for tailwind V4
```

Comment:
Should this be in the previous `if` block to prevent accidentally enabling tailwind because of transitive deps?

Reply:
Tailwind 4 doesn't have a co fig file anymore
It's all in css now